### PR TITLE
Script to invalidate vizjson redis cache #2930

### PR DESCRIPTION
**Pull request by rafatower**
@javisantana please review:
```
bundle exec rake cartodb:redis:purge_vizjson
```

### CODE REVIEW

File: lib/tasks/redis_maintainance.rake [hunk 1]
```diff
@@ -6,5 +6,15 @@ namespace :cartodb do
         user.save_metadata
       end
     end
+
+    desc "purge vizjson redis cache"
+    task :purge_vizjson => :environment do
+      User.each do |user|
```
Comment (javisantana) [inline lib/tasks/redis_maintainance.rake]: couldn't we just not iterate on users and just do all the visualizations ?

Comment (javisantana) [inline lib/tasks/redis_maintainance.rake]: and at least group DEL per user and only do a single one => http://redis.io/commands/del

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2675/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2679/))

**Comment by rafatower** [comment]
@javisantana can you please take another look?

**Comment by javisantana** [comment]
looks good if we don't reach max command limit in redis (if that even exists)

**Comment by rafatower** [comment]
I checked the doc: http://redis.io/commands/del

Also a small test in local:
```
u = User.where(username: 'development').first
redis_keys = CartoDB::Visualization::Collection.new.fetch(user_id: u.id).map(&:redis_vizjson_key)
fake_redis_keys = (1..50000).map(|i| "visualization:#{SecureRandom.uuid}:vizjson"}; nil
a_thilion_keys = redis_keys.concat fake_redis_keys
total_deleted = CartoDB::Visualization::Member.redis_cache.del a_thilion_keys
```
and it is ok. It will always compute more keys than actually stored, but I think that is also ok.

**Pull request merged**